Adyen/services.py

```diff
@@ -322,6 +329,42 @@ def payment_result(self, request=None, **kwargs):
         action = "paymentsResult"
         return self.client.call_checkout_api(request, action, **kwargs)
 
+    def payment_captures(self, path_param, request=None, idempotency_key=None, **kwargs):
+        if path_param == "":
+            raise ValueError(
+                'must contain a pspReference in the path_param, path_param cannot be empty'
+            )
+        action = "paymentsCapture"
+        return self.client.call_checkout_api(request, action, path_param, idempotency_key, **kwargs)
+
+    def payments_cancels_without_reference(self, request=None, idempotency_key=None, **kwargs):
+        action = "paymentsCancelsWithoutReference"
+        return self.client.call_checkout_api(request, action, idempotency_key, **kwargs)
+
+    def payments_cancels_with_reference(self, path_param, request=None, idempotency_key=None, **kwargs):
+        if path_param == "":
+            raise ValueError(
+                'must contain a pspReference in the path_param, path_param cannot be empty'
+            )
+        action = "paymentsCancelsWithReference"
+        return self.client.call_checkout_api(request, action, path_param, idempotency_key, **kwargs)
+
+    def payments_reversals(self, path_param, request=None, idempotency_key=None, **kwargs):
+        if path_param == "":
+            raise ValueError(
+                'must contain a pspReference in the path_param, path_param cannot be empty'
+            )
+        action = "paymentsReversals"
+        return self.client.call_checkout_api(request, action, path_param, idempotency_key, **kwargs)
```

Comment:
This does not work: `call_checkout_api` expects `idempotency_key` and `path_param` in a different order. 

Same comment probably also applies to all other uses of `call_checkout_api` in this PR.

Reply:
I see :) Thanks for the comment!